tools/cardano-sl-tools.cabal

```diff
@@ -140,13 +128,7 @@ executable cardano-wallet-hs2purs
   build-tools: cpphs >= 1.19
   ghc-options: -pgmP cpphs -optP --cpp
 
-  if flag(with-wallet) && flag(with-web)
-    buildable:         True
-  else
-    buildable:         False
-
-  if flag(with-web)
-    cpp-options: -DWITH_WEB
+  buildable:         False
```

Comment:
@gromakovsky Finally I have moved `cardano-wallet-hs2purs` into `cardano-sl-wallet` project. Thx again for pointing it out!